scalafix-sbt/src/main/scala/scalafix/sbt/ScalafixPlugin.scala

```diff
@@ -270,16 +270,12 @@ object ScalafixPlugin extends AutoPlugin {
             scalafixConfig.value
               .map(x => "--config" :: x.getAbsolutePath :: Nil)
               .getOrElse(Nil)
-          val ruleArgs =
-            if (inputArgs.nonEmpty)
-              inputArgs.flatMap("-r" :: _ :: Nil)
-            else Nil
+
```

Comment:
Before:
`my-project/scalafix RemoveUnusedImports`

After:
`my-project/scalafix -r RemoveUnusedImports`

Reply:
The whole motivation for having it work as it does currently was to avoid that pesky `-r` flag, but I agree it was adding way more complexity than it was worth. This was before --test and --diff and --config-str (and many other cli flags) so it made more sense back then, but I think it's good to make sbt-scalafix and scalafix-cli work the same 👍

Reply:
```
> scalafix -r Rem<TAB>
```
does not complete for me.

This is btw a pretty badly breaking change, potentially causing breaking people's CI where a .travis.yml might be written `sbt "scalafix -r Blah"`. Do you think we can keep the old behavior for rules while adding more flags?

Reply:
ah indeed, I removed the short flags because they block the autocompletion. I think it's possible to add it back in the parser without the completion.